xfce-extra/xfce4-cpugraph-plugin/files/gcc13.patch

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
You don't need to revbump for such a fix.

Reply:
Never mind, I dropped the revbumps.